### PR TITLE
updating the readme file with note

### PR DESCRIPTION
**Pull request by sjyothi54**
Hi @jaesius 
When I tried to integrate the snowflake integration in linux machine.I got the error like "$$NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux $$NEWRELIC_SNOWFLAKE_HOME/config.yaml $$NEWRELIC_SNOWFLAKE_HOME/queries/query_history.sql`
`/bin/sh: NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux: No such file or directory."
So I updated readme.md file with workaround.
This might fix some issues raised as well  in this repository.

### CODE REVIEW

**Comment by matewilk** [comment]
Hi @sjyothi54 

Thanks so much for contributing.

I checked the PR and it seems like some parts of the edited README are not visible, specifically:

> **Note:** 
If you are getting error like,
 `$$NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux $$NEWRELIC_SNOWFLAKE_HOME/config.yaml $$NEWRELIC_SNOWFLAKE_HOME/queries/query_history.sql`
`/bin/sh: NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux: No such file or directory.`

Would you be able to fix it before I merge?

**Comment by sjyothi54** [comment]
@matewilk Thanks for the response. Could you please elaborate on what I need to fix?

**Comment by nedl86** [comment]
Hi @sjyothi54 thanks so much for your attention to this! I think there's an alternate way to get past this error, by adding the following to your agent configuration (`newrelic-infra.yml` file) and restarting the agent.

```
passthrough_environment:
  - NEWRELIC_SNOWFLAKE_HOME
```

I wonder if this may be a simpler fix to add to the README?

**Comment by matewilk** [comment]
Hey @sjyothi54,

Sincere apologies for the late reply,
What I meant is that when you look at the `README.md` file in Github with your changes, the quoted text is not visible.
Is this intentional?
If not could you use [note](https://github.com/community/community/discussions/16925) notation instead?

**Comment by sjyothi54** [comment]
Sorry for the delayed response, @matewilk and @nedl86. 
I've implemented the recommendation from @nedl86, and it's functioning correctly on my end. Thank you for the advice. 
As a result, I have incorporated the modifications into this PR. Your review would be greatly appreciated. 
Thanks once again.

**Review by pkosanam — APPROVED** [review]
looks good to me.

Having this step on the readme file should help users in integrating snowflake with New Relic

File: README.md [hunk 1]
```diff
@@ -79,7 +79,14 @@ The integration relies on `NEWRELIC_SNOWFLAKE_HOME` environment variable to conn
 2. Replace the values in `snowflake.env`
 3. Edit the `newrelic-infra.service` service definition - `sudo nano /etc/systemd/system/newrelic-infra.service`
 4. Add a line `EnvironmentFile=/path/to/env/file` in the `[Service]` section
-5. Perform a daemon-reload and restart the newrelic-infra service - `sudo systemctl daemon-reload && sudo systemctl restart newrelic-infra`
+5. Edit the `newrelic-infra.yml` - `sudo nano /etc/newrelic-infra.yml`
+6. Add these lines in `newrelic-infra.yml`
+   ```
+   passthrough_environment:
+   - NEWRELIC_SNOWFLAKE_HOME
+   - SNOWSQL_ACCOUNT
```
Comment (pkosanam) [inline README.md:87]:
```suggestion
passthrough_environment:
  - NEWRELIC_SNOWFLAKE_HOME
  - SNOWSQL_ACCOUNT
```

**Pull request merged**